src/FixedDataTableCellGroup.js

```diff
@@ -192,14 +192,6 @@ class FixedDataTableCellGroup extends React.Component {
     zIndex: PropTypes.number.isRequired,
   }
 
-  shouldComponentUpdate(/*object*/ nextProps) /*boolean*/ {
```

Comment:
We still need this check and added parent's check (FixedDataTableRowImpl) doesn't cover it. 
This is because there can be multiple cell groups rendered by a row, and the scroll value can be different/same for each group (fixed cellgroups)

Reply:
yes agreed! Made changes